### PR TITLE
v0.21: Dumps

### PR DESCRIPTION
**Pull request by guimachiavelli**
- [x] reference/features/configuration.md: remove --dump-batch-size L#67, L#193, L#200, #L208
- [x] create/how_to/updating: remove --dump-batch-size L#241
- [x] reference/api/dump: remove --dump-batch-size L#11
- [x] reference/features/dumps: remove --dump-batch-size L#31
- [x] add startedAt field to POST /dumps route (reference/features/dumps, create/how_to/updating (L#220-227), reference/api/dump (L#49-54))
- [x] add startedAt and finishedAt fields to GET /dumps/:uid/status route (create/how_to/updating (L#204-212), reference/api/dump (L#25-30))
- [x] reference/features/dumps: update text so users know that only successfully completed dumps generate dump files and that a dump's status becomes unavailable after restarting an instance

### CODE REVIEW

File: create/how_to/updating.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ In this guide, we will:
 
 ### Step 1: Set all fields as displayed attributes
 
+::: note
+This step is not mandatory when creating dumps in MeiliSearch v0.21 and above.
+:::
+
```
Comment (curquiza) [inline create/how_to/updating.md:120]: this whole step was added due to a bug that is now fixed in v0.21.0 :)

Comment (curquiza) [reply create/how_to/updating.md:120]: I think you can completely delete this part

**Review by curquiza — CHANGES_REQUESTED** [review]

File: create/how_to/updating.md (hunk 1 above)
Comment (curquiza) [reply create/how_to/updating.md:120]: You don't need to put your attributes as displayed anymore.

Comment (guimachiavelli) [reply create/how_to/updating.md:120]: As discussed, we will keep this note and the first step as they are for users migrating from v0.20 to v0.21.

We will probably replace both with a very short note in a future release.

File: create/how_to/updating.md [hunk 2]
```diff
@@ -114,6 +114,10 @@ In this guide, we will:
 
 ### Step 1: Set all fields as displayed attributes
 
+::: note
+This step is not mandatory when creating dumps in MeiliSearch v0.21 and above.
```
Comment (curquiza) [inline create/how_to/updating.md:118]:
```suggestion
This step is not necessary if you created a dump using MeiliSearch v0.21.0 and later.
```

**Review by curquiza — CHANGES_REQUESTED** [review]

File: create/how_to/updating.md (hunk 2 above)
Comment (curquiza) [inline create/how_to/updating.md:118]: More than "not mandatory", it's completely useless and non-productive for the users. I would say "not necessary" or a synonym :)

**Review by CaroFG — APPROVED** [review]

File: create/how_to/updating.md (hunk 2 above)
Comment (CaroFG) [reply create/how_to/updating.md:118]: I'll even Write, `if your dump was created in MeiliSearch v0.21 or above, continue to step 2` or something similar

Comment (curquiza) [reply create/how_to/updating.md:118]: I like the Caro's suggestion

Comment (guimachiavelli) [reply create/how_to/updating.md:118]:
```suggestion
If your dump was created in MeiliSearch v0.21 or or above, continue to step 2.
```

**Comment by guimachiavelli** [comment]
@curquiza's comments were addressed, merging this into the v0.21 branch 🎊

**Pull request merged**